module/zfs/zvol.c

```diff
@@ -369,6 +369,41 @@ zvol_set_volsize(const char *name, uint64_t volsize)
 	return (SET_ERROR(error));
 }
 
+/*
+ * Update volthreading.
+ */
+int
+zvol_set_volthreading(const char *name, uint64_t value)
+{
+	zvol_state_t *zv = zvol_find_by_name(name, RW_NONE);
+	if (zv) {
+		zv->zv_threading = value;
+		mutex_exit(&zv->zv_state_lock);
+		return (0);
+	}
+	return (ENOENT);
+}
+
+/*
+ * Update zvol ro property.
+ */
+int
+zvol_prop_set_ro(const char *name, uint64_t value)
```

Comment:
For some reason out of all functions setting properties this seems the only one with "_prop_" in the name.